cmake/AddCTSOption.cmake

```diff
@@ -11,10 +11,12 @@ set(SYCL_CTS_DETAIL_OPTION_COMPILE_DEFINITIONS "")
 #            Use `#if <OPTION>` instead of `#ifdef <OPTION>`.
 #
 # Optional parameters:
-#  - WARN_IF_OFF <msg>     Print a warning message if this option is set to OFF.
+#  - WARN_IF_OFF <msg>    Print a warning message if this option is set to OFF.
+#  - FORCE_ON <value>     If <value> is ON the option will be overwritten as ON.
 #
 function(add_cts_option option_name option_description option_default)
     cmake_parse_arguments(PARSE_ARGV 3 args "" "WARN_IF_OFF" "")
+    cmake_parse_arguments(PARSE_ARGV 3 args "" "FORCE_ON" "")
```

Comment:
No need to parse args twice. Add `FORCE_ON` to the previous invocation. (`"WARN_IF_OFF;FORCE_ON"`) if we don't feed the option, one-value, multi-value args from variables.